test/gateway/pollingInterval.js

```diff
@@ -804,16 +794,6 @@ test('Polling schemas (should properly regenerate the schema when a downstream s
   })
 
   const refreshedSchema = `
-    directive @extends on INTERFACE | OBJECT
```

Comment:
It doesn't make sense for a service to have this defined in its schema. This is federation metadata.